ruby-monday-blog/spec/features/member_can_navigate_to_post_from_index_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.feature "member can navigate to post from post index", type: :feature do
+  before :each do
```

Comment:
* `:each` is the default action for `before` in Rspec 3
* `let` is generally prefered over using a `before` block, if you need it to be evaluated use `let!`